user-guide/docs/index.md

```diff
@@ -2,12 +2,14 @@
 
 ## User Account Registration
 
-Any natural hazards researcher or practitioner that wants an environment to store, analyze, curate, publish, and discover data with a community of peers may register for an account.
+Any natural hazards researcher or practitioner that wants an environment to store, analyze, curate, publish, and discover data with a community of peers may register for an account. Detailed account registration instructions can be found  [[here](https://docs.tacc.utexas.edu/basics/accounts/)].
```

Comment:
Let's drop the redundant syntax, otherwise user would see:

> [[here](https://docs.tacc.utexas.edu/basics/accounts/)]

```suggestion
Any natural hazards researcher or practitioner that wants an environment to store, analyze, curate, publish, and discover data with a community of peers may register for an account. Detailed account registration instructions can be found [here](https://docs.tacc.utexas.edu/basics/accounts/).
```